src/Pizzeria.hpp

```diff
@@ -27,5 +29,7 @@ class Pizzeria
 private:
     std::string name_;
     std::vector<Order> orders_;
+    std::unique_ptr<ITimer> timer_ptr = nullptr;
+    Timer timer_;
```

Comment:
as mentioned - why 2? what if timer_ptr is also of Timer type? Which one should be used?